tests/lib/rules/capitalized-comments.js

```diff
@@ -0,0 +1,1012 @@
+        },
+        {
+            code: "//* eslint-disable-line",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "//* eslint-disable-next-line",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n * eslint semi:off */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n * eslint-env node */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  istanbul ignore next */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  jshint asi:true */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  jscs: enable */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  global var1, var2 */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  global var1:true, var2 */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  globals var1, var2 */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  globals var1:true, var2 */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+        {
+            code: "/*\n *  exported myVar */",
+            options: ["always"],
+            errors: [{
+                message: ALWAYS_MESSAGE,
+                line: 1,
+                column: 1
+            }]
+        },
+
+        // ignoreInlineComments should only allow inline comments to pass
```

Comment:
Can you add a test to ensure that inline comments are *not* ignored if the option is omitted/set to false?